### PR TITLE
PP-946 Address review comments

### PR DESCRIPTION
**Pull request by kakumara**
## WHAT
Quick fix for address review comments from previous PR.

### CODE REVIEW

File: src/main/java/uk/gov/pay/api/filter/LoggingFilter.java [hunk 1]
```diff
@@ -29,8 +30,7 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
 
         String requestURL = ((HttpServletRequest) servletRequest).getRequestURI();
         String requestMethod = ((HttpServletRequest) servletRequest).getMethod();
-        Optional<String> requestIdMaybe = Optional.ofNullable(((HttpServletRequest) servletRequest).getHeader(HEADER_REQUEST_ID));
-        String requestId = requestIdMaybe.orElse("");
+        String requestId = StringUtils.defaultString(((HttpServletRequest)servletRequest).getHeader(HEADER_REQUEST_ID));
```
Comment (rahul-vinayak) [inline src/main/java/uk/gov/pay/api/filter/LoggingFilter.java]: now that you have made the change can you implement the same way at RestClientLoggingFilter:28

Comment (kakumara) [inline src/main/java/uk/gov/pay/api/filter/LoggingFilter.java]: done 👍

**Comment by kakumara** [comment]
rebuild

**Comment by kakumara** [comment]
rebuild

**Comment by simad** [comment]
rebuild

**Comment by simad** [comment]
rebuild

**Comment by rauligs** [comment]
rebuild

**Comment by rahul-vinayak** [comment]
👍

**Pull request merged**